daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "How-To: Invoke services using gRPC"
+linkTitle: "How-To: Invoke services using gRPC"
+description: "Call between services using service invocation"
+weight: 3000
+---
+
+This article describe how to deploy services each with an unique application ID, so that other services can discover and call endpoints on them using service invocation API.
+
```

Comment:
Add the community call video link
"Watch this video on how to call services using gRPC and Dapr." and add link to community call at this time https://youtu.be/B_vkXqptpXY?t=70

Reply:
Done